echopype/tests/visualize/test_plot.py

```diff
@@ -225,48 +225,48 @@ def test_water_level_echodata(water_level, expect_warning):
         ek_encode_mode=range_kwargs.get('encode_mode', 'power'),
     )
     single_array = range_in_meter.isel(frequency=0, ping_time=0).values
-    no_input_water_level = False
-    if isinstance(water_level, list):
-        water_level = water_level[0]
-        echodata.platform = echodata.platform.drop_vars('water_level')
-        no_input_water_level = True
-
-    if isinstance(water_level, xr.DataArray):
-        if 'frequency' in water_level.dims:
-            original_array = single_array + water_level.isel(frequency=0).values
-    elif isinstance(water_level, bool) and water_level is True:
-        if no_input_water_level is False:
+    no_input_range_offset = False
+    if isinstance(range_offset, list):
+        range_offset = range_offset[0]
+        echodata.platform = echodata.platform.drop_vars('range_offset')
+        no_input_range_offset = True
+
+    if isinstance(range_offset, xr.DataArray):
+        if 'frequency' in range_offset.dims:
+            original_array = single_array + range_offset.isel(frequency=0).values
+    elif isinstance(range_offset, bool) and range_offset is True:
+        if no_input_range_offset is False:
             original_array = (
                 single_array
-                + echodata.platform.water_level.isel(frequency=0, ping_time=0).values
+                + echodata.platform.range_offset.isel(frequency=0, ping_time=0).values
             )
         else:
             original_array = single_array
-    elif water_level is not False and isinstance(water_level, (int, float)):
-        original_array = single_array + water_level
+    elif range_offset is not False and isinstance(range_offset, (int, float)):
+        original_array = single_array + range_offset
     else:
         original_array = single_array
 
     results = None
     try:
         if expect_warning:
             with pytest.warns(UserWarning):
-                results = _add_water_level(
+                results = _add_range_offset(
                     range_in_meter=range_in_meter,
-                    water_level=water_level,
+                    range_offset=range_offset,
                     data_type=EchoData,
                     platform_data=echodata.platform,
                 )
         else:
-            results = _add_water_level(
+            results = _add_range_offset(
                 range_in_meter=range_in_meter,
-                water_level=water_level,
+                range_offset=range_offset,
                 data_type=EchoData,
                 platform_data=echodata.platform,
             )
     except Exception as e:
         assert isinstance(e, ValueError)
-        assert str(e) == 'Water level must have any of these dimensions: frequency, ping_time, range_bin'  # noqa
+        assert str(e) == 'Range offset must have any of these dimensions: frequency, ping_time, range_bin'  # noqa
```

Comment:
Copying @leewujung's comment from elsewhere: seems like `Range offset` should be `range_offset` since it is a variable name?